src/igvfd/schemas/changelogs/measurement_set.md

```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 12
 
 * Add `release_timestamp`.
+* Expand `preferred_assay_title` enum list to include `ATAC-seq`, `RNA-seq`, `TF ChIP-seq`, `histone ChIP-seq`, `snRNA-seq`, `scRNA-seq`, `snATAC-seq` and `scATAC-seq`.
```

Comment:
please change changelog to be: " Add `ATAC-seq`,..... to `preferred_assay_title`."